src/gui/qml/credentials/BasicAuthCredentials.qml

```diff
@@ -46,6 +46,9 @@ Credentials {
         Keys.onBacktabPressed: {
             widget.parentFocusWidget.focusPrevious();
         }
+
```

Comment:
We are using a layout and must not mix anchors with layouts.

Reply:
done